### ISSUE TITLE
`Web3 provider for using Dapps` in brave://settings/extensions is not translated

### ISSUE THREAD

**Issue by btlechowski** [brave/brave-browser#12013]
This is a regression in 1.5.71, Not reproducible in 1.5.70.

## Steps to Reproduce

1. Run Brave in Polish locale
1. Open `brave://settings/extensions`

## Actual result:

`Web3 provider for using Dapps` is not translated

![image](https://user-images.githubusercontent.com/34715963/95199425-15369e80-07dd-11eb-8bff-75e814109bd2.png)

## Expected result:
`Web3 provider for using Dapps` is translated
![image](https://user-images.githubusercontent.com/34715963/95199365-018b3800-07dd-11eb-877b-30215f27f532.png)

## Reproduces how often: 

Easily reproduced

## Brave version (brave://version info)

Brave | 1.15.71 Chromium: 86.0.4240.72 (Official Build) (64-bit)
-- | --
Revision | 581582174c512f44f44fd1aea340471f54b2365f-refs/branch-heads/4240@{#1134}
OS | Ubuntu 18.04 LTS

cc @brave/legacy_qa @mkarolin @rebron

**Comment by LaurenWags** [comment]
Reproduced on macOS using French and German with

```

Brave | 1.15.71 Chromium: 86.0.4240.72 (Offizieller Build) (x86_64)
-- | --
Überarbeitung | 581582174c512f44f44fd1aea340471f54b2365f-refs/branch-heads/4240@{#1134}
Betriebssystem | macOS Version 10.14.6 (Build 18G3020)

```

**Comment by kjozwiak** [comment]
Closing and moved into https://github.com/brave/brave-browser/milestone/135 as https://github.com/brave/brave-core/pull/6788 has been merged into `1.15.x` 👍

**Comment by LaurenWags** [comment]
Verified passed with

```
Brave | 1.15.72 Chromium: 86.0.4240.75 (Offizieller Build) (x86_64)
-- | --
Überarbeitung | c69c33933bfc72a159aceb4aeca939eb0087416c-refs/branch-heads/4240@{#1149}
Betriebssystem | macOS Version 10.14.6 (Build 18G3020)
```

Verified STR from description for both French and German:

**`French`**
<img width="986" alt="12013 French" src="https://user-images.githubusercontent.com/28145373/95338632-22a56480-0881-11eb-9edd-f51a6b98ab7e.png">

**`German`**
<img width="993" alt="12013 German" src="https://user-images.githubusercontent.com/28145373/95338647-26d18200-0881-11eb-95d0-ca80d625cd12.png">

----------
